pkgs/by-name/bo/bookstack/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  fetchFromGitHub,
+  php,
+  nixosTests,
+  nix-update-script,
+  dataDir ? "/var/lib/bookstack",
+}:
+
+php.buildComposerProject2 (finalAttrs: {
+  pname = "bookstack";
+  version = "25.02";
+
+  src = fetchFromGitHub {
+    owner = "bookstackapp";
+    repo = finalAttrs.pname;
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-Aj9R9+jXqq2Th76kfED5ZfZaMyFyK6c8r4LwXGMomL0=";
+  };
+
+  vendorHash = "sha256-VFGCV8g6ph3TckpJQMk0g/F256Mz0t0VFW52jzBfmm0=";
+
+  passthru = {
+    phpPackage = php;
+    tests = nixosTests.bookstack;
+    updateScript = nix-update-script { };
```

Comment:
When you have a composer.lock in a repo it's not necessary to add updateScript as it's already added by the builder https://github.com/NixOS/nixpkgs/blob/9a151383e21aa90de05d246e3c9e6587e0f36ecc/pkgs/build-support/php/builders/v2/build-composer-project.nix#L108